pkg/templates/templates.go

```diff
@@ -13,12 +15,13 @@ var basicFunctions = template.FuncMap{
 		a, _ := json.Marshal(v)
 		return string(a)
 	},
-	"split": strings.Split,
-	"join":  strings.Join,
-	"title": strings.Title,
-	"lower": strings.ToLower,
-	"upper": strings.ToUpper,
-	"pad":   padWithSpace,
+	"split":   strings.Split,
+	"join":    strings.Join,
+	"title":   strings.Title,
+	"lower":   strings.ToLower,
+	"upper":   strings.ToUpper,
+	"pad":     padWithSpace,
+	"shorten": stringid.TruncateID,
```

Comment:
Since this works on any string, I think it should take an integer argument to shorten to a certain length